staging/src/k8s.io/apiserver/pkg/server/egressselector/egress_selector.go

```diff
@@ -150,22 +150,22 @@ type grpcProxier struct {
 	tunnel client.Tunnel
 }
 
-func (g *grpcProxier) proxy(addr string) (net.Conn, error) {
+func (g *grpcProxier) proxy(addr, caller string) (net.Conn, error) {
 	return g.tunnel.Dial("tcp", addr)
```

Comment:
So on the GRPC connection we are just dropping the caller info on the floor? (for now)

Reply:
This was from a limitation from the way we set up the http connect and grpc tunnels. http connect propagates the user agent on the PROXY function while grpc propagates the user agent on the CONNECT function. L202 captures this propagation for grpc. Since the interface needs to be consistent between the two implementations, I thought it'd be appropriate to pass in the caller as a parameter to both the connect and proxy function and have one of the functions perform a noop.